### ISSUE TITLE
Could not load file or assembly on Windows 2012 r2

### ISSUE THREAD

**Issue by itguyscl** [DevExpress/AjaxControlToolkit#51]
Hello,
[AJAX-error.txt](https://github.com/DevExpress/AjaxControlToolkit/files/202488/AJAX-error.txt)
I am moving a web sites from Windows 2008 IIS7 to Windows 2012 R2 IIS8.5 and I run into an issue with AJAX Control Toolkit on Windows 2012 R2 server.
I have attached the error logged in Application log of the server.
I have AjaxControlToolkit.dll from May, 2010, in the Bin folder of the web site. I already checked Bin folder and Temporary ASP.NET Files folder permissions.
What else should I check to make this work?
Please advise.
Thank you!

**Comment by MikhailTymchukDX** [comment]
@itguyscl Please check to see you load AjaxControlToolkit.dll into the IIS Application Pool with a proper version of .NET Framework: http://stackoverflow.com/a/13519900/644496 
Also, make sure the `Enable 32-bit Applications` property is set to `true`: http://www.codeproject.com/Tips/325824/Running-a-bit-application-under-IIS-on-a-bit

**Comment by itguyscl** [comment]
Thank you for your response.
Yes, both conditions are true (please see attached)
![app-pool](https://cloud.githubusercontent.com/assets/18263641/14254891/b6085b04-fa5f-11e5-8a79-858fb39e2e45.jpg)

**Comment by itguyscl** [comment]
I installed the latest version of AJAXControlToolkit and copied the DLL file into the Bin folder on the web site. It seems to work now as I can see the reference to that DLL in the "Temporary ASP .Net Files" folder, but I get an error about "Unknown server tag 'asp:ToolkitScriptManager'", as to my understanding ToolkitScriptManager has been removed from version 15.1 and later. Any quick suggestions on how to fix this?
